### PR TITLE
add cases GH and GL integrations and drift stuff

### PR DESCRIPTION
**Pull request by misterpantz**
#### Description

Adds GitHub cases
Adds GitLab cases
Updates all cases content
Add drift
Add aggregation
Add space settings

#### Related issue

none

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [x] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: docs/platform/cases/manage.mdx [hunk 1]
```diff
@@ -68,19 +68,80 @@ import Partial from "../partials/_editor-owner.mdx";
 
 :::note
 
-Closing an ticket or issue in your ticket system or does not close the corresponding case in Mondoo. If you create cases using a direct Jira or Zendesk integration, a space-wide setting controls whether closing a case in Mondoo automatically closes the corresponding issue in Jira. To learn more, read the section below.
+Closing an ticket or issue in your ticket system or does not close the corresponding case in Mondoo. If you create cases using a direct integration with your ticket system, a space-wide setting controls whether closing a case in Mondoo automatically closes the corresponding issue/ticket in your ticket system. To learn more, read the section below.
```
Comment (tas50) [inline docs/platform/cases/manage.mdx:71]: Is this "ticket system" or "ticketing system"?

File: docs/platform/cases/manage.mdx [hunk 2]
```diff
@@ -68,19 +68,80 @@ import Partial from "../partials/_editor-owner.mdx";
...
 
 :::
 
-### Choose whether to closes tickets/issues when you close cases
+## Automatically create cases on drift
+
+Drift occurs when, instead of improving security, an asset becomes _more_ vulnerable to attack:
```
Comment (tas50) [inline docs/platform/cases/manage.mdx:77]: Moving forward should probably be pushing terms like "more exposed"

File: docs/platform/cases/manage.mdx [hunk 3]
```diff
@@ -68,19 +68,80 @@ import Partial from "../partials/_editor-owner.mdx";
...
+
+<Partial />{" "}
+
+1. Navigate to the space where you want to change the drift setting.
+
+2. In the navigation sidebar, select **Settings** and then select the **Cases** tab.
+
+3. Enable or disable **Automatically create cases on drift**.
+
+
```
Comment (tas50) [inline docs/platform/cases/manage.mdx:99]:
```suggestion
```

File: docs/platform/cases/manage.mdx (hunk 2 above)
Comment (misterpantz) [reply docs/platform/cases/manage.mdx:77]: I can change this now.

**Review by tas50 — APPROVED** [review]

**Pull request merged**